base/ordering.jl

```diff
@@ -18,9 +18,24 @@ export # not exported by Base
     DirectOrdering,
     lt, ord, ordtype
 
+"""
```

Comment:
Better show the fully qualified name as it's not exported:
```suggestion
"""
        Base.Order.Ordering

```

Same elsewhere.